src/main/fc/boot.c

```diff
@@ -742,6 +742,22 @@ void configureScheduler(void)
 
     if (sensors(SENSOR_ACC)) {
         setTaskEnabled(TASK_ACCEL, true);
+        switch (gyroPeriodUs) {  // Switch statement kept in place to change acc rates in the future
```

Comment:
yeah, i noted this on the BF code, I think we talked about it before, just about to clean that up now.